staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/metrics.go

```diff
@@ -52,7 +52,7 @@ var (
 		prometheus.CounterOpts{
 			Namespace: namespace,
 			Subsystem: subsystem,
-			Name:      "rejectedRequests",
+			Name:      "rejected_requests",
```

Comment:
This should according to our instrumentation guideline also have the _total suffix.